dev-support/docker/Dockerfile

```diff
@@ -51,11 +51,14 @@ RUN apt-get -q update \
         gcc \
         git \
         gnupg-agent \
+        hugo \
```

Comment:
Good that we got rid of the explicit installation of boost, hugo and protobuf.

Reply:
nodejs too. 😁